packages/toolpad-components/src/Autocomplete.tsx

```diff
@@ -113,6 +159,7 @@ export default createComponent(Autocomplete, {
       helperText: 'If true, the autocomplete will be disabled.',
       type: 'boolean',
     },
+    ..._.pick(FORM_INPUT_ARG_TYPES, ['name', 'isRequired', 'minLength', 'maxLength', 'isInvalid']),
```

Comment:
I don't we should add the cognitive and computational overhead of lodash for this use case. I'd either do

```tsx
const FORM_INPUT_ARG_TYPES = {
  // name, isRequired, isInvalid
}

const FORM_TEXT_INPUT_ARG_TYPES = {
  // minLength, maxLength
}

{
// ...
  ...FORM_INPUT_ARG_TYPES,
  ...FORM_TEXT_INPUT_ARG_TYPES
}
```

or

```tsx
const FORM_INPUT_ARG_TYPES = {
  // name, isRequired, isInvalid
}

const FORM_TEXT_INPUT_ARG_TYPES = {
  ...FORM_INPUT_ARG_TYPES,
  // minLength, maxLength
}

{
// ...
  ...FORM_TEXT_INPUT_ARG_TYPES
}
```